examples/blueprint-construct/index.ts

```diff
@@ -23,7 +23,7 @@ export interface BlueprintConstructProps {
 export default class BlueprintConstruct {
     constructor(scope: Construct, props: cdk.StackProps) {
 
-        blueprints.HelmAddOn.validateHelmVersions = true;
+        blueprints.HelmAddOn.validateHelmVersions = false;
```

Comment:
was it intentional?

Reply:
No it is not. Used it for my testing. Reverted back